NOTICE

```diff
@@ -0,0 +1,24 @@
+Apache Baremaps (incubating)
+Copyright 2022-2023 The Apache Software Foundation.
+
+Baremaps development team
+Copyright 2020-2022.
+
+This product includes protocol buffers definitions copied from OSMPBF.
+Copyright (c) 2010 Scott A. Crosby. <scott@sacrosby.com>
+MIT License.
+Please visit the following URLs for the licensing information:
+https://github.com/openstreetmap/OSM-binary/issues/18
+https://github.com/openstreetmap/OSM-binary/pull/35
```

Comment:
Since it uses the MIT License, you don't need to modify the NOTICE.

Reply:
As the discussion regarding the licensing of OSMPBF is a bit scattered on the web (the original version in the osmosis project has not been updated by Scott Crosby), I wanted to keep track of it.